composer.json

```diff
@@ -107,6 +107,7 @@
         }
     },
     "require": {
+        "ext-json": "*",
```

Comment:
Maybe better to lock this down to a minor version at least?

Reply:
Agree here, let's lock this down

Reply:
Doesn't work